.gitignore

```
@@ -1,11 +1,51 @@
-target/
-pom.xml.tag
-pom.xml.releaseBackup
-pom.xml.versionsBackup
-pom.xml.next
-release.properties
+# use glob syntax.
+syntax: glob
+*.ser
+*.class
+*~
+*.bak
+#*.off
+*.old
+.DS_Store
+
+# eclipse conf file
+.settings
+.classpath
+.manager
+.scala_dependencies
+
+# VsCode
+.project
+
+# idea
+.idea
+*.iml
+
+# building
+target
+build
+null
+tmp*
+temp*
+dist
+test-output
+build.log
+
+# other scm
+.svn
+.CVS
+.hg*
+
+# switch to regexp syntax.
+#  syntax: regexp
+#  ^\.pc/
+
+#SHITTY output not in target directory
```

```suggestion
output not in target directory
```

I blame @felipemmelo

---

😆  Omg, Thanks!

---

Sorry my mistake

---

No worries